tools/ktx/encode_utils_basis.h

```diff
@@ -318,40 +268,22 @@ struct OptionsCodec {
                 cxxopts::value<float>(), "<deviation>")
             (kUastcRdoF, "Do not favor simpler UASTC modes in RDO mode.")
             (kUastcRdoM, "Disable RDO multithreading (slightly higher compression, deterministic).");
-        opts.add_options("Encode common")
-            (kNormalMode, "Optimizes for encoding textures with normal data. If the input texture has "
-                "three or four linear components it is assumed to be a three component linear normal "
-                "map storing unit length normals as (R=X, G=Y, B=Z). A fourth component will be ignored. "
-                "The map will be converted to a two component X+Y normal map stored as (RGB=X, A=Y) "
-                "prior to encoding. If unsure that your normals are unit length, use --normalize. "
-                "If the input has 2 linear components it is assumed to be an X+Y map of unit normals.\n"
-                "The Z component can be recovered programmatically in shader code by using the equations:\n"
-                "    nml.xy = texture(...).ga;              // Load in [0,1]\n"
-                "    nml.xy = nml.xy * 2.0 - 1.0;           // Unpack to [-1,1]\n"
-                "    nml.z = sqrt(1 - dot(nml.xy, nml.xy)); // Compute Z\n"
-                "ETC1S / BasisLZ encoding, RDO is disabled (no selector RDO, no endpoint RDO) to provide better quality.")
-            (kThreads, "Sets the number of threads to use during encoding. By default, encoding "
-                "will use the number of threads reported by thread::hardware_concurrency or 1 if "
-                "value returned is 0.", cxxopts::value<uint32_t>(), "<count>")
-            (kNoSse, "Forbid use of the SSE instruction set. Ignored if CPU does "
-               "not support SSE. SSE can only be disabled on the basis-lz and "
-               "uastc compressors.");
     }
 
-    EncodeCodec validateEncodeCodec(const cxxopts::OptionValue& codecOpt) const {
-        static const std::unordered_map<std::string, EncodeCodec> codecs = {
-            { "basis-lz", EncodeCodec::BasisLZ },
-            { "uastc", EncodeCodec::UASTC }
+    BasisCodec validateEncodeCodec(const cxxopts::OptionValue& codecOpt) const {
```

Comment:
`validateBasisCodec` since the return type is now `BasisCodec`?

Reply:
https://github.com/KhronosGroup/KTX-Software/pull/817/commits/93f6d1feec6bf9ab04dd0a00cbbb94fac6bd2983